google/cloud/internal/backoff_policy.h

```diff
@@ -126,44 +126,44 @@ class ExponentialBackoffPolicy : public BackoffPolicy {
    *     for more details.
    */
   template <typename Rep1, typename Period1, typename Rep2, typename Period2>
-  ExponentialBackoffPolicy(std::chrono::duration<Rep1, Period1> initial_delay,
+  ExponentialBackoffPolicy(std::chrono::duration<Rep1, Period1> minimum_delay,
```

Comment:
Thanks for updating the name. Can we also update the `@p initial_delay`s in the documentation?